vaadin-testbench-core-junit5/pom.xml

```diff
@@ -32,6 +32,7 @@
 
     <properties>
         <junit5.version>5.9.1</junit5.version>
+        <junit5.platform.version>1.9.1</junit5.platform.version>
```

Comment:
If you import the junit5-bom the second property is not needed and correctly managed by junit5

Reply:
Good point, updated. The bom name for junit5 is `junit-bom` - bit confusing.